test/format/config.c

```diff
@@ -285,6 +286,25 @@ config_table(TABLE *table, void *arg)
             config_single(table, "btree.value_min=20", false);
     }
 
+    /*
+     * Limit the rows to 1000000 if the realloc exact and realloc malloc configs are on and not all
+     * explicitly set. Realloc exact config allocates the exact amount of memory, which causes a new
+     * realloc call every time we append to an array. Realloc malloc turns a single realloc call to
+     * a malloc, a memcpy, and a free. The combination of both will significantly slow the
+     * execution.
+     */
+    if ((!config_explicit(NULL, "debug.realloc_exact") ||
+          !config_explicit(NULL, "debug.realloc_malloc") || !config_explicit(NULL, "runs.rows") ||
+          config_range(NULL, "runs.rows")) &&
+      GV(DEBUG_REALLOC_EXACT) && GV(DEBUG_REALLOC_MALLOC) &&
+      NTV(tables[0], RUNS_ROWS) > WT_MILLION) {
+        config_single(tables[0], "runs.rows=1000000", false);
```

Comment:
I think this needs to be true as per my previous comments.

Reply:
We only set it to true if it is set from a config.